uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/mocks/data/entitlement-data.js

```diff
@@ -0,0 +1,141 @@
+//contains data for the mock api calls for entitlement
+define(function() {
+  return {
+    all_data: {
```

Comment:
yes, but this is a response json from server which uses snake case and hence the naming convention